hbase_sql/src/test/java/com/splicemachine/stream/StreamListenerIT.java

```diff
@@ -0,0 +1,102 @@
+package com.splicemachine.stream;
+
+import com.splicemachine.derby.test.framework.*;
+import com.splicemachine.test.LongerThanTwoMinutes;
+import com.splicemachine.test.SerialTest;
+import com.splicemachine.test.SlowTest;
+import org.junit.*;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.*;
+
+@Category({LongerThanTwoMinutes.class, SlowTest.class, SerialTest.class})
+public class StreamListenerIT {
+
+    private static final String CLASS_NAME = StreamListenerIT.class.getSimpleName().toUpperCase();
+    private static final SpliceWatcher spliceClassWatcher = new SpliceWatcher(CLASS_NAME);
+    private static final SpliceSchemaWatcher schemaWatcher = new SpliceSchemaWatcher(CLASS_NAME);
+    protected static final SpliceTableWatcher A_TABLE = new SpliceTableWatcher("A",schemaWatcher.schemaName,
+            "(a1 int)");
+    private static final int TABLE_SIZE = 1572864;
+
+    private ExecutorService executor = Executors.newFixedThreadPool(4);
+
+    @ClassRule
+    public static TestRule chain = RuleChain.outerRule(spliceClassWatcher)
+            .around(schemaWatcher)
+            .around(A_TABLE)
+            .around(new SpliceDataWatcher() {
+                @Override
+                protected void starting(Description description) {
+                    try {
+                        spliceClassWatcher.execute("insert into a values 1,2,3");
+                        try (PreparedStatement ps = spliceClassWatcher.prepareStatement("insert into a select a1 + (select count(*) from a) from a")) {
+                            for (int i = 0; i < 19; i++) {
+                                ps.execute();
+                            }
+                        }
+                    } catch (Exception e) {
+                        throw new RuntimeException(e);
```

Comment:
I guess you can remove the `catch` since it is useless and leave `finally`.

Reply:
This test is sporadic, I've removed it and will address it in another task